NetSDK/App.config

```diff
@@ -36,6 +36,9 @@
     <add key="TEST_IMPRESSIONS_URL_TEMPLATE" value="testImpressions/bulk" />
     <add key="METRICS_URL_TEMPLATE " value="metrics/{endpoint}" />
     <add key="SPLITS_STORAGE_CONCURRENCY_LEVEL" value="4" />
+    <add key="TEST_IMPRESSIONS_REFRESH_RATE" value=""/>
+    <add key="TEST_IMPRESSIONS_CACHE_SIZE_PER_TEST" value=""/>
+    <add key="EVENTS_BASE_URL" value=""/>
```

Comment:
let's add a default here: `https://events.split.io` or where ever it makes sense.